tapir/shifts/templates/shifts/shift_detail.html

```diff
@@ -23,7 +23,7 @@
 {% endblock head %}
 {% block content %}
     <div class="card mb-2" id="shift_detail_card">
-        <div class="card-header">
+        <div class="card-header d-flex flex-wrap">
```

Comment:
That doesn't look correct on desktop: 
![image](https://github.com/user-attachments/assets/e67f63f1-d830-4de7-8635-fa34daee26e3)